packages/@o3r/schematics/src/rule-factories/ng-add/dependencies.ts

```diff
@@ -170,7 +170,7 @@ export const setupDependencies = (options: SetupDependenciesOptions): Rule => {
     const ngAddToRun = new Set(Object.keys(options.dependencies)
       .filter((dep) => options.ngAddToRun?.some((pattern) => typeof pattern === 'string' ? pattern === dep : pattern.test(dep))));
     const requiringInstallList = new Set(Object.entries(options.dependencies).filter(([, { requireInstall }]) => requireInstall).map(([dep]) => dep));
```

Comment:
This doesn't seem to be used anymore, we can probably change it to something like:
```suggestion
    const isInstallRequired = Object.values(options.dependencies).some(({ requireInstall }) => requireInstall);
```